File: kotlin/11c.kt
fun main(){
    val userAns: Int = readln().toIntOrNull() ?: 0;
    val userAns2: Int = readln().toIntOrNull() ?: 0;
    if(userAns > userAns2){
        println("${userAns} is greater than ${userAns2}");
    } else if(userAns < userAns2){
        println("${userAns} is less than ${userAns2}");
    } else {
        println("${userAns} is equal to ${userAns2}");
    }
}

File: js/11c.js
//11c
const prompt = require("prompt-sync")({ sigint: true });

let userAns = prompt();
let userAns2 = prompt();

if (userAns>userAns2){
    console.log(`${userAns} is greater than ${userAns2}`);
}
else if (userAns<userAns2){
    console.log(`${userAns} is less than ${userAns2}`);
}
else {
    console.log(`${userAns} is equal to ${userAns2}`);
}
